topology.yaml

```diff
@@ -27,6 +27,12 @@ build args:
         metavar: ver
 
 start args:
+    --license-credentials:
+        help: MapR license credentials to use in the format username:password
+        metavar: credentials
+    --license-path:
```

Comment:
If it's a URL, perhaps call this `--license-url` to avoid confusion and change the `metavar` to `url`? In that case, you'd also need to update the README help text